build.ps1

```diff
@@ -14,6 +14,7 @@ $sdkFile = Join-Path $solutionPath "global.json"
 $libraryProjects = @(
     (Join-Path $solutionPath "src" "JustSaying" "JustSaying.csproj"),
     (Join-Path $solutionPath "src" "JustSaying.Models" "JustSaying.Models.csproj"),
+    (Join-Path $solutionPath "src" "JustSaying.Extensions.DependencyInjection.AwsCore" "JustSaying.Extensions.DependencyInjection.AwsCore.csproj"),
```

Comment:
I like this PR, it's a very handy addition. Everything is looking good-to-go, I'm just not happy yet with the package name.

I'm struggling with the name mostly because our naming convention is already a mouthful and the package from AWS is badly named, so I don't think we have an obvious nice option.

The `NETCore` part of the upstream AWS library is now out-of-date, so should just be `Net`, but `Net` in namespaces typically means network, so the AWS package should probably just be named `AWSSDK.Extensions.Setup` in hindsight.

I was thinking of something like:
* `JustSaying.Extensions.DependencyInjection.AwsExtensions`
or given that it's not a typical DI package, and it's just piggy-backing off of the `.Microsoft` package, perhaps we just go with:
* `JustSaying.AwsExtensionsSetup`

I'm not too excited by those suggestions, so would love to hear other options.

Reply:
Of the two options above, I think I'd go with `JustSaying.Extensions.DependencyInjection.AwsExtensions` to fit with the other two extension packages.